### ISSUE TITLE
formatter removes {/unless}

### ISSUE THREAD

**Issue by simonmcconnell** [surface-ui/surface_formatter#26]
before

```elixir
  @impl true
  def render(assigns) do
    ~F"""
    <div class="flex flex-col w-screen h-screen">
      <div class="flex-grow w-full bg-white overflow-auto">
        <div class="h-full max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
          <div class="h-full py-6 flex flex-col justify-center items-center">
            <div class="relative py-6 sm:max-w-7xl sm:mx-auto">
              <div class="absolute inset-0 bg-gradient-to-r from-blue-700 to-blue-800 shadow-lg transform -skew-y-6 sm:skew-y-0 sm:rotate-2 sm:rounded-none" />
              <div class="relative bg-white border border-blue-800 shadow-lg rounded-none">
                {!--{#unless @step == 0}
                  <div class="absolute left-1 top-2 h-12 w-12">
                    <img src="/images/logo.svg" class="h-12 w-12">
                  </div>
                {/unless}--}
                {#case @step}
                  {#match 0}
                    {render_welcome()}
                  {#match 1}
                    <div class="max-w-none mx-auto">
                      <div class="bg-white overflow-hidden">
                        {render_card_header("Display & Reporting")}
                        <TimeAndDateSettings id="time-and-date-settings" {=@start_of_day} {=@timezone} />
                      </div>
                    </div>
                  {#match 2}
                    <div class="max-w-none mx-auto">
                      <div class="bg-white overflow-hidden">
                        {render_card_header(
                          "Default Alarm Columns",
                          "Choose the columns displayed on the alarm log and alarm search results screen."
                        )}
                        <div class="px-6 py-2">
                          <InitDefaultAlarmColumns id="default-alarm-columns" drop_zone={Settings.get(:default_alarm_columns)} />
                        </div>
                      </div>
                    </div>
                  {#match 3}
                    <div class="max-w-none mx-auto">
                      <div class="bg-white overflow-hidden">
                        {render_card_header(
                          "Alarm Categories",
                          "Label and order alarm categories.  Categories can be defined as a combination of single numbers, lists of numbers or ranges.  There can be no overlap between categories.  For example, '1', '100-199', '1, 2, 5' and '10, 100-199, 1001' are all valid. To avoid importing entire categories of alarms, uncheck LOG."
                        )}
                        <div class="px-6 py-2">
                          <AlarmCategoriesSettings id="alarm-categories-settings" show_sidebar={@live_action == :edit_alarm_category} />
                        </div>
                      </div>
                    </div>
                  {#match _}
                    {render_done()}
                {/case}
              </div>
            </div>
          </div>
        </div>
      </div>
      {#unless @step == 0}
        <div class="flex items-center justify-end p-4 w-full bg-gray-50">
          <Button click="change_step" values={step: @step - 1}>
            Back
          </Button>

          <Button color="green" click="change_step" values={step: @step + 1} left_margin>
            {#if @step < @steps}
              Next
            {#else}
              Done
            {/if}
          </Button>
        </div>
      {/unless}
    </div>
    """
  end
```

run `mix surface.format` and it removes the `{/unless}` from the second last line.  The line stays there, but is empty :(

after: 

``` elixir
  @impl true
  def render(assigns) do
    ~F"""
    <div class="flex flex-col w-screen h-screen">
      <div class="flex-grow w-full bg-white overflow-auto">
        <div class="h-full max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
          <div class="h-full py-6 flex flex-col justify-center items-center">
            <div class="relative py-6 sm:max-w-7xl sm:mx-auto">
              <div class="absolute inset-0 bg-gradient-to-r from-blue-700 to-blue-800 shadow-lg transform -skew-y-6 sm:skew-y-0 sm:rotate-2 sm:rounded-none" />
              <div class="relative bg-white border border-blue-800 shadow-lg rounded-none">
                {!--{#unless @step == 0}
                  <div class="absolute left-1 top-2 h-12 w-12">
                    <img src="/images/logo.svg" class="h-12 w-12">
                  </div>
                {/unless}--}
                {#case @step}
                  {#match 0}
                    {render_welcome()}
                  {#match 1}
                    <div class="max-w-none mx-auto">
                      <div class="bg-white overflow-hidden">
                        {render_card_header("Display & Reporting")}
                        <TimeAndDateSettings id="time-and-date-settings" {=@start_of_day} {=@timezone} />
                      </div>
                    </div>
                  {#match 2}
                    <div class="max-w-none mx-auto">
                      <div class="bg-white overflow-hidden">
                        {render_card_header(
                          "Default Alarm Columns",
                          "Choose the columns displayed on the alarm log and alarm search results screen."
                        )}
                        <div class="px-6 py-2">
                          <InitDefaultAlarmColumns id="default-alarm-columns" drop_zone={Settings.get(:default_alarm_columns)} />
                        </div>
                      </div>
                    </div>
                  {#match 3}
                    <div class="max-w-none mx-auto">
                      <div class="bg-white overflow-hidden">
                        {render_card_header(
                          "Alarm Categories",
                          "Label and order alarm categories.  Categories can be defined as a combination of single numbers, lists of numbers or ranges.  There can be no overlap between categories.  For example, '1', '100-199', '1, 2, 5' and '10, 100-199, 1001' are all valid. To avoid importing entire categories of alarms, uncheck LOG."
                        )}
                        <div class="px-6 py-2">
                          <AlarmCategoriesSettings id="alarm-categories-settings" show_sidebar={@live_action == :edit_alarm_category} />
                        </div>
                      </div>
                    </div>
                  {#match _}
                    {render_done()}
                {/case}
              </div>
            </div>
          </div>
        </div>
      </div>
      {#unless @step == 0}
        <div class="flex items-center justify-end p-4 w-full bg-gray-50">
          <Button click="change_step" values={step: @step - 1}>
            Back
          </Button>

          <Button color="green" click="change_step" values={step: @step + 1} left_margin>
            {#if @step < @steps}
              Next
            {#else}
              Done
            {/if}
          </Button>
        </div>
      
    </div>
    """
  end
```

workaround: replace with `{#if ...}`

**Comment by paulstatezny** [comment]
Thanks for reporting this @simonmcconnell! Fixed in #33. I'll post back here when a patch is published.

**Comment by paulstatezny** [comment]
@simonmcconnell The fix was published just now in v0.5.1: https://hex.pm/packages/surface_formatter/0.5.1
